src/frontends/tensorflow/src/decoder_proto.cpp

```diff
@@ -101,8 +101,11 @@ ov::Any DecoderProto::get_attribute(const std::string& name) const {
     case ::tensorflow::AttrValue::ValueCase::kI:
         return attrs[0].i();
     case ::tensorflow::AttrValue::ValueCase::kShape: {
-        std::vector<ov::Dimension> dims;
         const auto& tf_shape = attrs[0].shape();
+        if (tf_shape.unknown_rank()) {
+            return ov::PartialShape::dynamic();
+        }
+        std::vector<ov::Dimension> dims;
```

Comment:
nitpick:
```
std::vector<ov::Dimension> dims(tf_shape.dim_size());
for (auto& dim : dims) {
    dim = tf_shape.dim(i).size();
  }
```

Reply:
Fixed in https://github.com/openvinotoolkit/openvino/pull/14238